### PR TITLE
PSR-2

### PR DESCRIPTION
**Pull request by roenschg**
Changed code style to fit the PSR-2 standard (http://www.php-fig.org/psr/psr-2/).

### CODE REVIEW

**Comment by roenschg** [comment]
Pull Request also fixes an recently discovered issue on a fcgi environment. https://github.com/roenschg/ShibbolethBundle/issues/1

File: Service/Shibboleth.php [hunk 1]
```diff
@@ -22,52 +22,55 @@
...
+        'originSite'    => array('header'=> 'Shib-Origin-Site', 'server' => 'originSite', 'multivalue'=> false),
+        'authenticationInstant' => array('header'=> 'Shib-Authentication-Instant', 'server' => 'authenticationInstant', 'multivalue' => false),
+        'employeeType' => array('header'=> 'Shib-KUL-employeeType', 'server' => 'employeeType', 'multivalue'=> false),
+        'studentType' => array('header'=> 'Shib-KUL-studentType', 'server' => 'studentType', 'multivalue'=> true),
+        'primouNumber' => array('header'=> 'Shib-KUL-PrimouNumber', 'server' => 'primouNumber', 'multivalue'=> true),
+        'ouNumber' => array('header'=> 'Shib-KUL-ouNumber', 'server' => 'ouNumber', 'multivalue'=> true),
+        'dipl' => array('header'=> 'Shib-KUL-dipl', 'server' => 'dipl', 'multivalue'=> true),
+        'opl' => array('header'=> 'Shib-KUL-opl', 'server' => 'opl', 'multivalue'=> true),
+        'campus' => array('header'=> 'Shib-KUL-campus', 'server' => 'campus', 'multivalue'=> false),
+        'logoutURL' => array('header' => 'Shib-logoutURL', 'server' => 'Shib-logoutURL', 'multivalue' => false)
```
Comment (vierbergenlars) [inline Service/Shibboleth.php]: This key is present twice in the array, and holds an incorrect `server` value

**Comment by roenschg** [comment]
@vierbergenlars The default naming might fit in the most cases but it might happen (like in my case) that you have to overwrite names.
I will change the server variables so that an update won´t change anything if you don´t overwrite the variables in the configuration file.

**Comment by rmoreas** [comment]
@roenschg thanks for the contribution. I'll review it as soon as possible.

**Pull request merged**